hyperliquid/exchange_plus.py

```diff
@@ -0,0 +1,106 @@
+from eth_account.signers.local import LocalAccount
+
+from hyperliquid.info import Info
+from hyperliquid.exchange import Exchange
+from hyperliquid.utils.types import Any, List, Literal, Meta, Optional, Tuple, Cloid
+
+# Extends Exchange Class with 'Market Orders'
+class ExchangePlus(Exchange):
```

Comment:
This seems like a small enough addition to the state, that it would make more sense to just add it to Exchange instead of making a new class

Reply:
np ... I would commit all suggested changes in one batch once you have decided:
- do we round sizes for the user
- does the user need to pass mid_price or can the class fetch the price by itself if needed

Reply:
Let's remove the automatic rounding of sizes but keep the code to fetch the price as needed